parsl/tests/test_serialization/test_3495_deserialize_managerlost.py

```diff
@@ -0,0 +1,47 @@
+import os
+import signal
+
+import pytest
+
+import parsl
+from parsl import Config, HighThroughputExecutor
+
+
+@parsl.python_app
+def get_manager_pgid():
+    import os
+    return os.getpgid(os.getpid())
+
+
+@parsl.python_app
+def lose_manager():
+    import os
+    import signal
+
+    manager_pid = os.getppid()
+    os.kill(manager_pid, signal.SIGSTOP)
+
+
+@pytest.mark.local
+def test_manager_lost_system_failure(tmpd_cwd):
+    hte = HighThroughputExecutor(
+        label="htex_local",
+        address="127.0.0.1",
+        max_workers_per_node=2,
+        cores_per_worker=1,
+        worker_logdir_root=str(tmpd_cwd),
+        heartbeat_period=1,
+        heartbeat_threshold=1,
+    )
+    c = Config(executors=[hte], strategy='simple', strategy_period=0.1)
+
+    with parsl.load(c):
+        manager_pgid = get_manager_pgid().result()
+        try:
+            lose_manager().result()
+        except Exception as e:
```

Comment:
i guess here I would have put in a test that what comes back really is a ManagerLost

Reply:
Yeah; I see that.  That was actually the first iteration of my test, but I lost the thread yesterday when sifting through the "why won't it work?!" details.  I'll look at a follow-up PR later today.

Reply:
Addressed in #3497.